.changes/v3.11.0/1123-improvements.md

```diff
@@ -0,0 +1 @@
+* Add property `upload_any_file` to resource `vcd_catalog_media` to allow uploading any file as catalog media item [GH-1123]
```

Comment:
Should we put original PR #1123 or this one? (#1124)

Reply:
PR #1123 implemented the resource change. This one (built on top of that branch) implemented the data source change. I'd leave it as it is. If I have to make changes, I'll put both numbers